### PR TITLE
Tests for version, use conda-build for download/check

### LINKED ISSUE #12: latest 2.45 windows builds contains 2.41 binary
OSX and Linux appear to be fine.

Guess we need to check the version in the tests...

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Fixes #12. Halfway to #1, actually, as this proposes using the conda-build machinery to download the raw binaries, and manage the shasums directly. Have also added more aggressive tests that do even more `$PATH` hacking to ensure it does what is says on the tin.

Also adds myself as a maintainer, as I seem to be bumping up on this package fairly frequently.

### CODE REVIEW

**Comment by bollwyvl** [comment]
Sorry about all the noise on there: looks like we have happy builds now :)

File: recipe/download-chromedriver.py [hunk 1]
```diff
@@ -1,21 +0,0 @@
-import os
```
Comment (igortg) [inline recipe/download-chromedriver.py]: :fire: :fire: :+1:

File: recipe/run_test.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import subprocess
```
Comment (igortg) [inline recipe/run_test.py]: nice

**Comment by bollwyvl** [comment]
Hooray, thanks for the speedy review!

**Comment by bollwyvl** [comment]
Eek,

https://travis-ci.org/conda-forge/python-chromedriver-binary-feedstock/jobs/479544384

Luckily it's Travis, and probably just needs a re-kick... I'll get to it as
soon as I can, but have intermittent network right now...

**Comment by bollwyvl** [comment]
Rekicked that, good to go!

**Pull request merged**